### PR TITLE
Issue #12898: changed type of PI from int to double

### PR DESCRIPTION
**Pull request by jxr98**
Issue #12898
I changed the type of PI from int to double in the [documentation](https://checkstyle.org/config_coding.html#VariableDeclarationUsageDistance).

### CODE REVIEW

**Comment by nrmancuso** [comment]
@jxr98 please grep by `int PI` to see if we copy-pasted this elsewhere in documentation

**Comment by jxr98** [comment]
> @jxr98 please grep by `int PI` to see if we copy-pasted this elsewhere in documentation

Done.
image

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -312,7 +312,7 @@ public final class TokenTypes {
      *
      * <p>For example:</p>
      * <pre>
-     * final int PI = 3.14;
+     * final double PI = 3.14;
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:315]: `--LITERAL_INT`

This line below should be changed .

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java (hunk 1 above)
Comment (jxr98) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:315]: Oh right. Thanks.

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**